src/collections/Menu/Menu.js

```diff
@@ -34,7 +34,7 @@ export default class Menu extends Component {
       const activeItemName = !hasActiveItem && i === 0
         ? child.props.name
         : this.state.activeItem || this.props.activeItem;
-      return React.cloneElement(child, {
+      return child && React.cloneElement(child, {
```

Comment:
The menu component cannot clone the child if it doesn't exist.  When filtering the doc site menu by search we we're getting errors.  Now, we skip cloning and updating menu children if the child is falsy.